docs/sphinx/source/changelog/pending.rst

```diff
@@ -0,0 +1,13 @@
+*************************
+Pending
+*************************
+
+Deprecations
+------------
+* The the deprecation of :py:func:`~rdtools.normalization.pvwatts_dc_power`,
+  and :py:func:`~rdtools.normalization.normalize_with_pvwatts` has been reversed.
+  (:pull:'225')
```

Comment:
This is #227

Reply:
Also needs backticks on the `:pull:`, not singlequotes

Reply:
Good catches, thanks!